Client/src/main/java/com/bbd/BeanClient/ClientApplication.java

```diff
@@ -51,13 +51,13 @@ public CommandLineRunner run(RestTemplate restTemplate) throws Exception {
 
             // Running tests
             try {
-                boolean beanResult = banBean(1, true);
```

Comment:
This change seems to keep bouncing around in multiple commits. Could you merge in main once more (sorry) and have a look a these? Surely there's no need for removing createComment, breaking the else up, etc.

Reply:
Yea, Im using an auto formatter . I will leave createComment removed for now, the changes in clientapplication are relatively arbitrary since its used to test , but moving forth I will just paste my created function.